backend/controllers/projectCtrls.js

```diff
@@ -1,7 +1,69 @@
 const db = require("../models");
 
 // TODO ROUTE FUNCTIONS (NOT ACTIONS) GO HERE
+const getProject = (req, res) => {
+    db.Project.find({})
+    .then((foundProject) => {
+        if(!foundPeople) {
+            res.status(404).json({message: "Cannont find project."})
+        } else {
+            res.status(200).json({data: foundProject})
+        }
+    });
+};
+
+
+const createProject = (req, res) => {
+    req.body.password = bcrypt.hashSync(
+    req.body.password,
+    bcrypt.genSaltSync(10)
+);
+db.Project.create(req.body, (err, createdProject) => {
+    if (err) {
+        res.send(500, err);
+    } else if (!createdProject) {
+        res.send(400);
+    } else {
+        console.log("Project Added", createdProject);
+        res.send(200, createProject);
+    }
+});
+};
+
+const updateProject = (req, res) => {
+    if (req.body.password) {
+        req.body.password = bcrypt.hashSync(
+            req.body.password,
+            bcrypt.genSaltSync(10) 
+        );
+    }
+    db.Project.findByIdAndUpdate(req.session.currentUser._id, req.body, { new: true }).then((updatedProject) => {
```

Comment:
This doesn't work. You're trying to find a project's id by its user's ID. 

1. Take out encryption, no passwords should be sent here
2. Grab the project by *its* ID from the database (should be req.params.id), then check its user id against the user id from req.session. If that's the same proceed if not send 403 forbidden